src/gc/gc.d

```diff
@@ -1350,6 +1350,17 @@ immutable uint[B_MAX] binsize = [ 16,32,64,128,256,512,1024,2048,4096 ];
 immutable size_t[B_MAX] notbinsize = [ ~(16-1),~(32-1),~(64-1),~(128-1),~(256-1),
                                 ~(512-1),~(1024-1),~(2048-1),~(4096-1) ];
 
+alias PAGEBITS = GCBits.wordtype[PAGESIZE / 16 / GCBits.BITS_PER_WORD];
+static assert(PAGEBITS.length * GCBits.BITS_PER_WORD * 16 == PAGESIZE);
+
+void set(ref PAGEBITS bits, size_t i) @nogc pure nothrow
+in { assert(i < PAGEBITS.sizeof * 8); }
```

Comment:
I suggest we move away from this bulky idiom. All you need is place the `assert` at the beginning of the function.